### ISSUE TITLE
Our app needs a web server

### ISSUE THREAD

**Issue by hshayesteh** [dambileh/Civitas#6]
Depends on #5 

We decided to use Nginx https://www.nginx.com/blog/5-performance-tips-for-node-js-applications/

This ticket is about installing Nginx and getting it to direct traffic to our Gateway

**Comment by hadiosbourne** [comment]
We will put this on hold as its not very critical, we can have a look at using google endpoint instead https://cloud.google.com/endpoints/
